package.json

```diff
@@ -53,5 +53,6 @@
   ],
   "publishConfig": {
     "access": "public"
-  }
+  },
```

Comment:
whats the upside of this? just means everyone needs to upgrade/downgrade their pnpm

Reply:
it squeezed in the PR somehow :) I'll remove it